config/application.rb

```diff
@@ -3,7 +3,26 @@
 end
 require File.expand_path('boot', __dir__)
 require 'apipie/middleware/checksum_in_headers'
-require 'rails/all'
+
+# Only import the frameworks we need - this list is taken from rails/all
+require "rails"
+
+[
+  'active_record/railtie',
+  # 'active_storage/engine',
+  'action_controller/railtie',
+  'action_view/railtie',
+  'action_mailer/railtie',
+  'active_job/railtie',
+  # 'action_cable/engine',
+  # 'action_mailbox/engine',
+  # 'action_text/engine',
+  'sprockets/railtie',
+].each do |railtie|
+  require railtie
+rescue LoadError
+end
+require 'rails/test_unit/railtie' if Rails.env.test?
```

Comment:
Should this live in `test/test_helper.rb`?

Reply:
let's try and see :)

Reply:
Looks like it doesn't work - this is the railtie that adds the test tasks to rake, i'll change it to only load when not in production.